panel/layout/modal.py

```diff
@@ -0,0 +1,187 @@
+"""The `Modal` is an element that displays in front of and deactivates all other page content."""
+import param
+
+from ..reactive import ReactiveHTML
+from .base import NamedListLike
+
+# See https://a11y-dialog.netlify.app/
+JS_FILE = "https://cdn.jsdelivr.net/npm/a11y-dialog@7/dist/a11y-dialog.min.js"
```

Comment:
- [ ] I should try upgrading to v8.

Reply:
With v8 microsoft explorer is no longer supported. Do we support microsoft explorer @philippjfr ?